packages/framework/attributor/README.md

```diff
@@ -1,4 +1,84 @@
 # @fluid-internal/attributor
 
-This package contains code for framework support for attribution of content.
-It's currently in early development and should not be relied on for any production code; backwards compatibility is not yet guaranteed.
+This package contains definitions and implementations for framework-provided attribution functionality.
+
+## Quickstart
+
+To turn on op-stream based attribution in your container, use `mixinAttributor` to create a `ContainerRuntime` class which supports querying for attribution information.
+
+```typescript
+import { ContainerRuntime } from "@fluidframework/container-runtime";
+
+const ContainerRuntimeWithAttribution = mixinAttributor(ContainerRuntime);
+
+// ...then, in your ContainerRuntime factory use this class:
+class ContainerRuntimeFactory implements IRuntimeFactory {
+    public async instantiateRuntime(context: IContainerContext, existing?: boolean): Promise<IRuntime> {
+        const runtime = await ContainerRuntimeWithAttribution.load(
+            /* same arguments you would have provided to ContainerRuntime.load */
+        );
+        // do whatever setup is necessary with the runtime here
+        return runtime;
+    }
+}
+```
+
+TODO: document any settings you need to turn on to get DDSes to track attribution
```

Comment:
will come back to clean up this TODO once this is solved. going to file some short work for it in-sprint splitting off documentation task.